docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -154,16 +154,41 @@ belongs.
   The data type of this field must be categorical.
   
 `predicted_field`::
-  (Required, string) The field in the `index` that contains the predicted value, 
+  (Optional, string) The field in the `index` which contains the predicted value,
   in other words the results of the {classanalysis}.
 
+`predicted_class_field`::
```

Comment:
> predicted_class_field

We can fix this in a subsequent PR if necessary, but this option doesn't appear in the right order alphabetically.

Reply:
Done.
But this makes me think how will the user understand the difference between `predicted_field` and `predicted_class_field`. The former is non-nested (and used for precision, recall, etc.) while the latter is nested (and used for AucRoc).
Do you have an idea on how to express the difference clearly?